opensrp-chw/src/kk/java/org/smartregister/chw/interactor/AncHomeVisitInteractorFlv.java

```diff
@@ -969,4 +982,66 @@ public void onPayloadReceived(BaseAncHomeVisitAction baseAncHomeVisitAction) {
             Timber.v("onPayloadReceived");
         }
     }
+
+    private class ChwObservationsAction implements BaseAncHomeVisitAction.AncHomeVisitActionHelper {
+        private String anyone_presence;
+        private Context context;
+
+        @Override
+        public void onJsonFormLoaded(String s, Context context, Map<String, List<VisitDetail>> map) {
+            this.context = context;
+        }
+
+        @Override
+        public String getPreProcessed() {
+            return null;
+        }
+
+        @Override
+        public void onPayloadReceived(String jsonPayload) {
+            try {
+                JSONObject jsonObject = new JSONObject(jsonPayload);
+                anyone_presence = JsonFormUtils.getCheckBoxValue(jsonObject, "anyone_present");
+            } catch (JSONException e) {
+                Timber.e(e);
+            }
+        }
+
+        @Override
+        public BaseAncHomeVisitAction.ScheduleStatus getPreProcessedStatus() {
+            return null;
+        }
+
+        @Override
+        public String getPreProcessedSubTitle() {
+            return null;
+        }
+
+        @Override
+        public String postProcess(String s) {
+            return null;
+        }
+
+        @Override
+        public String evaluateSubTitle() {
+            return MessageFormat.format(context.getString(R.string.chw_observations_evaluate_sub_title), anyone_presence);
+        }
+
+        @Override
+        public BaseAncHomeVisitAction.Status evaluateStatusOnPayload() {
+            if (!StringUtils.isBlank(anyone_presence))
+                if(anyone_presence.equalsIgnoreCase("yes") || anyone_presence.equalsIgnoreCase("ndio")){
```

Comment:
Try and avoid using checkbox values to check for something but instead use the the key which is always the same, the values can change someone can change the translations from "ndio" to "ndiyo" and your condition here will fail. You can have two variables one that holds the checkbox value in case you want to show that in a subtitle but another one that holds the checkbox key that you can use in your conditions